app/src/lib/git/status.ts

```diff
@@ -202,18 +202,25 @@ export async function getStatus(
   }
 
   const mergeHeadFound = await isMergeHeadSet(repository)
+  const conflictedFilesInIndex = entries.some(e => e.statusCode === 'UU')
   const rebaseInternalState = await getRebaseInternalState(repository)
 
   if (enablePullWithRebase()) {
     conflictDetails = await getConflictDetails(
       repository,
       mergeHeadFound,
+      conflictedFilesInIndex,
       rebaseInternalState
     )
     // a little optimization so we don't run `getWorkingDirectoryConflictDetails`
     // when there are no files to even detect conflicts in
   } else if (entries.length > 0) {
```

Comment:
i think we can remove this tiny optimization now since looking through the entries should cover that case